src/index.js

```diff
@@ -69,10 +69,10 @@ ${unused.join(`\n`)}`);
   }
 
   _getFileDepsMap(compilation) {
-    const fileDepsBy = compilation.fileDependencies.reduce((acc, usedFilepath) => ({
-      ...acc,
-      [usedFilepath]: usedFilepath,
-    }), {});
+    const fileDepsBy = compilation.fileDependencies.reduce((acc, usedFilepath) => {
+      acc[usedFilepath] = usedFilepath;
```

Comment:
Maybe switching to `_.set(acc, usedFilepath, usedFilepath)` could just skipping the eslint error